### PR TITLE
[android_intent] added support for more than 70th actions

### PR DESCRIPTION
**Pull request by pedromassango**
## Description

I added support for most of the Settings's actions based on Android documentation: https://developer.android.com/reference/android/provider/Settings

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Comment by collinjackson** [comment]
@pedromassango Thanks for the contribution. If you'd like us to land this, could you please address the reviewer feedback when you get a chance?

File: packages/android_intent/README.md [hunk 1]
```diff
@@ -29,6 +29,15 @@ for it in the plugin and use an action constant to refer to it. For instance:
 
 `'action_location_source_settings'` translates to `android.settings.LOCATION_SOURCE_SETTINGS`
 
+This plugin brings a set of actions defined on the Android side so that you can call most of the android actions from this plugin as:
```
[the remark this answers is not in the document]
Comment (pedromassango) [reply packages/android_intent/README.md]: I agree. I'll be pushing new commits in this weekend

**Comment by pedromassango** [comment]
> @pedromassango Thanks for the contribution. If you'd like us to land this, could you please address the reviewer feedback when you get a chance?

@collinjackson sorry I didn't get your point

**Comment by pedromassango** [comment]
Any update on this?

**Comment by nerder** [comment]
This is very useful work and should not be loosed 👍 Any chance this can be merged at some point?

**Comment by pedromassango** [comment]
I'm still not sure how to test this.

**Comment by stuartmorgan** [comment]
Thanks for the submission! However, we’re currently not accepting non-critical PRs for this plugin, as we hope that in time we’re able to transition users to the corresponding plugin in [the Flutter Community Plus Plugins repository](https://plus.fluttercommunity.dev/). We encourage you to submit your PR there.

(We apologize for the long delay in triaging this PR. We’re in the process of overhauling our PR triage system to respond much more quickly, as well as working through the backlog.)

**Pull request closed without merging**